oauth2/oauth2-jose/src/main/java/org/springframework/security/oauth2/jwt/JwtClaimValidator.java

```diff
@@ -61,6 +65,9 @@ public OAuth2TokenValidatorResult validate(Jwt token) {
 		if (test.test(claimValue)) {
 			return OAuth2TokenValidatorResult.success();
 		} else {
+			if (logger.isDebugEnabled()) {
+				logger.debug("Failed to validate OAuth2 token claim " + claim);
```

Comment:
It'd be nice to have the `error` description align with the debug message here, too, like you did with `nbf`.